Makefile

```diff
@@ -14,16 +14,26 @@ all: $(OUTPUT).html $(OUTPUT).txt $(OUTPUT).xml
 	$(info RFC rendering has been tested with mmark version 2.1.1 and xml2rfc 2.30.0, please ensure these are installed and recent enough.)
 
 $(OUTPUT).md: specification.markdown rfc_frontmatter.markdown rfc_backmatter.markdown EBMLSchema.xsd ebml_schema_example.xml
-	cat rfc_frontmatter.markdown $< rfc_backmatter.markdown > $(OUTPUT).md
+	cat rfc_frontmatter.markdown $< rfc_backmatter.markdown | sed "s/@BUILD_DATE@/$(shell date -I)/" > $(OUTPUT).md
 
 %.xml: %.md
 	$(MMARK) $< > $@
 	sed -i -e 's/<sourcecode type="xsd">/<sourcecode type="xml">/' $@
 
-%.html: %.xml
+rfc8794.xml: $(OUTPUT).xml
+	sed -e 's/<?xml version="1.0" encoding="utf-8"?>/<?xml version="1.0" encoding="UTF-8"?>\n<!DOCTYPE rfc SYSTEM "rfc2629-xhtml.ent">/' \
+	-e "s/docName=\"8794\"/docName=\"$(OUTPUT)\"/" \
+	-e 's@<organization></organization>@@' \
+	-e 's@<street></street>@@' \
+	-e 's@BCP 14@BCP\&nbsp;14@' \
+	-e 's@<dd><t>@<dd>@' \
+	$< | \
+	awk 1 RS='</t>\n</dd>' ORS='</dd>' | head -n -1 > $@
```

Comment:
Hmm, same issue, works with gnu head but not macOS head.

Reply:
I have to find a way to run a macOS shell. This kind of discrepancies can be detected in bootstrap and adjust the commands.

Reply:
I don't think that that needs to block the PR. I can see that it works and hope to replace the head and sed with a process that works well natively on mac.

Reply:
OK. I tried with FreeBSD but still have to come up with a proper solution (otherwise a better way to remove the `</t>` from `</t>\n</dd>` than awk would be better). xsltproc is also not an option as it will mess the newlines in the text (I think)